### ISSUE TITLE
Change the Firefox version of the addon to simply enable the built-in addon.

### ISSUE THREAD

**Issue by wisniewskit** [webcompat/webcompat-reporter-extensions#154]
Desktop and Android Firefox ship with a more feature-rich copy of the webcompat reporter, though it is disabled by default on release builds. As such we could simply have this add-on flip the pref to enable that version (`extensions.webcompat-reporter.enabled`).

It's unclear whether it will be considered kosher by the addons team to whitelist that specific pref for this specific addon, but I think a good enough case can be made to do so (less code to review in this add-on on AMO, less duplication in code, etc).

**Comment by gamer191** [comment]
Just wondering, how can one use ``extensions.webcompat-reporter.enabled``? I tried enabling it and it did nothing. Does it require an extension to act as a GUI for it?

**Comment by wisniewskit** [comment]
@gamer191, yes, to make it easier to ship updated versions out-of-sync with the main Firefox update process, and to access more helpful capabilities than normal web extensions can access, we ship a version of the webcompat reporter and webcompat interventions features as bundled ("system") addons on desktop and Android Firefox versions. That's why the prefs are named that way.

That said, that version of the addon should be bundled with desktop and Android Firefox, even if it's disabled by default except on pre-release versions. So if you're not seeing the Help menu as "Report Site Issue" appear after you toggle that preference, we would need to do some troubleshooting to figure out why that might be.

**Comment by gamer191** [comment]
> So if you're not seeing the Help menu as "Report Site Issue" appear after you toggle that preference, we would need to do some troubleshooting to figure out why that might be.

Thanks, I do see that. Apologies, I missed it
